src/IdentityAuthClient.js

```diff
@@ -14,7 +14,10 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+import * as Matrix from 'matrix-js-sdk';
```

Comment:
it's a bit unfortunate that we pull in the whole package here, it makes future tree-shaking have to do a bit more work.

Reply:
Ah right, I guess cargo-culted that from other places... I'll refine it here.